app/controllers/legacy_tasks_controller.rb

```diff
@@ -47,9 +49,15 @@ def user_role
     (params[:role] || user.vacols_roles.first).try(:downcase)
   end
 
-  def redirect_using_normalized_css_id
+  def use_css_id
+    params[:user_id] = User.find(params[:user_id]).css_id
```

Comment:
As we have already passed the `verify user exists step here, we can simply use
```suggestion
    params[:user_id] = user.css_id
```